libs/state/src/checkout/order/reducers/order.reducer.ts

```diff
@@ -0,0 +1,29 @@
+import { OrderActions, OrderActionTypes } from '../actions/order.actions';
+import { Cart } from '@daffodil/core';
+
+export interface State {
+  order: Cart,
```

Comment:
This isn't a "cart" its now an "order".

Reply:
I had misunderstood you when we were talking about "copying" over the cart on a place order. Added order model